.github/pull_request_template.md

```diff
@@ -6,5 +6,5 @@ If this pull request addresses an issue on GitHub, make sure to reference that
 issue using one of the
 [supported keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
 
-Before creating a pull request, make sure to comply with
-[`Contributing Guidelines`](/CONTRIBUTING.md).
+Before creating a pull request, make sure to comply with the
+[Contributing Guidelines](https://github.com/nginx/nginx/blob/master/CONTRIBUTING.md).
```

Comment:
```suggestion
[Contributing Guidelines](./CONTRIBUTING.md).
```
a relative link would be better imo, so it always references the same revision when browsing older ones

Reply:
I agree, but _will that work?_ GitHub seems really inconsistent with how it handles paths in Markdown links across different parts of the repo.

(E.g., the rendered diff for this patch handles `/CONTRIBUTING.md` differently from how it handles it in PR bodies, as you can see in #123.)

I spot checked this against a few other major open source projects ([nixpkgs](https://raw.githubusercontent.com/NixOS/nixpkgs/master/.github/PULL_REQUEST_TEMPLATE.md), [homebrew](https://github.com/Homebrew/brew/blob/master/.github/PULL_REQUEST_TEMPLATE.md)) and they all seem to use fully expanded URLs.

Reply:
You are correct, it does not. Sorry.
I didn't realize this was a pull request template.
I probably shouldn't review random PRs I stumble over while being tired :sweat_smile:

Reply:
Thanks for the review either way! It's good to get this written down :)